defaults/main.yml

```diff
@@ -2,6 +2,8 @@
 
 # --- Defaults (Welcome to override!) ---
 packagecontrol_url: "https://packagecontrol.io/Package Control.sublime-package"
+packagecontrol_dir_bin: "~/.config/sublime-text-3/Installed\ Packages"
```

Comment:
Not sure that slash needed inside quotes here, in README.md and tasks/main.yml.
WDYT?